### PR TITLE
Updates to the modelling resources

### PR DESCRIPTION
**Pull request by JGrinst**
Added details about Crop Covers not previously included here.

Removed links to downloading the TIF files as Crop Cover is a premium source.

(edit by john: you can see the resulting documentation here:
https://developers.gro-intelligence.com/JGrinst-patch-1/modeling-resources.html
vs. the current documentation:
https://developers.gro-intelligence.com/modeling-resources.html )

### CODE REVIEW

File: docs/modeling-resources.rst [hunk 1]
```diff
@@ -101,16 +71,79 @@ Since India does not have the equivalent of NASS CDL available to the public, we
...
+
+A low-confidence cover was made from the yearly crop covers starting in 2014. The results of these covers can be viewed in Gro under the Land Cover (percent) metric, where the value of a pixel will represent that pixel's likelihood of growing this crop in a given year.
+ 
+Brazil Soy
+----------
+Brazil soybeans uses a similar methodology to that of Argentina soybeans, with the exception that Landsat 5 was omitted from the constellation of satellites.
+
+Another minor change: Instead of subtracting the vegetation hue layer from the soil hue layer and isolating the top portion of the pixel values as was done for Argentina, , for Brazil we threshold and extract the soil values in each of the soil images and combine them with the other satellite soil imagery. The same is done with the vegetation values over all the vegetation images. This small technique change proved to be much more useful in finding crop areas, as it allowed for areas with poor coverage to be picked up more easily and not be drowned out.
```
Comment (doriapenciucgro) [inline docs/modeling-resources.rst]: extra space and comma here `Argentina, ,`

File: docs/modeling-resources.rst [hunk 2]
```diff
@@ -101,16 +71,79 @@ Since India does not have the equivalent of NASS CDL available to the public, we
...
+
+A low-confidence cover and a high-confidence cover were made from those yearly crop covers starting in 2009.  Each of them was further clumped to remove erroneous pixels. The results of these covers can be viewed in Gro under the Land Cover (percent) metric, where the value of a pixel will represent that pixel's likelihood of growing this crop in a given year.
```
Comment (doriapenciucgro) [inline docs/modeling-resources.rst]: Two spaces here before `  Each`

File: docs/modeling-resources.rst [hunk 3]
```diff
@@ -101,16 +71,79 @@ Since India does not have the equivalent of NASS CDL available to the public, we
...
+
+The crop discrimination methods used were different for different areas studied. One method used Sentinel-2 bands in a normalized difference of band 7 (~0.783 µm) and band 5 (~0.705 µm): B7 - B5/B7 + B5. The other method was based on band 8 (~0.842 µm), band 7 (~0.783 µm), band 6 (~0.74 µm), and band 5 (~0.705 µm), expressed as: B7 - B6/B8 + B5.
+
+We also eliminated pixels that were on a slope greater than 5°. Crude forest covers based on thresholding annual NDVI as well as SWIR bands (~1.61 µm) and NIR bands (~0.865 µm) were also applied to the regions. 
+
+We also made two types of covers for different data sources: NBS (low-confidence cover only) and Cofeed (high- and low-confidence covers). The results of these covers can be viewed in Gro under the Land Cover (percent) metric, where the value of a pixel will represent that pixel's likelihood of growing this crop in a given year.
```
Comment (doriapenciucgro) [inline docs/modeling-resources.rst]: high- confidence?

**Review by doriapenciucgro — CHANGES_REQUESTED** [review]

File: docs/modeling-resources.rst (hunk 1 above)
Comment (JGrinst) [reply docs/modeling-resources.rst]: Good spot! Fixed.

File: docs/modeling-resources.rst (hunk 2 above)
Comment (JGrinst) [reply docs/modeling-resources.rst]: Fixed!

File: docs/modeling-resources.rst (hunk 3 above)
Comment (JGrinst) [reply docs/modeling-resources.rst]: Added a "confidence"

**Review by doriapenciucgro — APPROVED** [review]

**Pull request merged**